goals/serializers.py

```diff
@@ -4,13 +4,13 @@
 
 class GoalSerializer(serializers.ModelSerializer):
     included_serializers = {
-        'usergoal_set': "goals.serializers.UserGoalSerialier",
+        'usergoal': "goals.serializers.UserGoalSerialier",
```

Comment:
won't it be user_goal now? as per the latest casing used?

Reply:
Not sure about this. Haven't seen any JSON:API spec example for using snake case here.